airbyte-integrations/connectors/destination-bigquery-denormalized/src/test/java/io/airbyte/integrations/destination/bigquery/util/TestBigQueryDenormalizedRecordFormatter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.bigquery.util;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.airbyte.integrations.destination.StandardNameTransformer;
+import io.airbyte.integrations.destination.bigquery.formatter.DefaultBigQueryDenormalizedRecordFormatter;
+import io.airbyte.protocol.models.AirbyteRecordMessage;
+
+public class TestBigQueryDenormalizedRecordFormatter extends
+    DefaultBigQueryDenormalizedRecordFormatter {
+
+  public TestBigQueryDenormalizedRecordFormatter(
+                                                 JsonNode jsonSchema,
+                                                 StandardNameTransformer namingResolver) {
+    super(jsonSchema, namingResolver);
+  }
+
+  @Override
+  public void addAirbyteColumns(ObjectNode data,
+                                AirbyteRecordMessage recordMessage) {
+    super.addAirbyteColumns(data, recordMessage);
```

Comment:
why do we created this class if overwritten method only call super implementation?

Reply:
This is a test class is located in the test package and exposes a protected method from the main package.
@VisibleForTesting annotation doesn't work here, I didn't want to use reflection to keep code more readable.